src/vellum/workflows/nodes/displayable/inline_prompt_node/node.py

```diff
@@ -46,18 +46,38 @@ def run(self) -> Iterator[BaseOutput]:
             )
 
         string_outputs = []
+        json_outputs = []
+
+        should_parse_json = False
+        if hasattr(self, "parameters"):
+            custom_params = getattr(self.parameters, "custom_parameters")
+            if custom_params and isinstance(custom_params, dict):
+                json_schema = custom_params.get("json_schema", {})
```

Comment:
[nit] `custom_params.get("json_mode", {})` is the other case we want to do this for